packages/media-sound/audacious-plugins/audacious-plugins.exlib

```diff
@@ -88,7 +90,7 @@ DEPENDENCIES="
             dev-libs/libcdio[>=0.70]
             media-libs/libcddb[>=1.2.1]
         )
-        cue? ( media-libs/libcue )
+        cue? ( media-libs/libcue[>=2.0] )
         dbus? ( dev-libs/dbus-glib:1[>=0.60] )
         fluidsynth? ( media-sound/fluidsynth[>=1.0.6] )
         ffmpeg? ( media/ffmpeg[>=0.9.1] )
```

Comment:
https://github.com/audacious-media-player/audacious-plugins/commit/5f6843cb5567db59d0a76c54585dc471b8833caa raised the minimum FFmpeg version, it's always a bit of a hassle to find out the right version but according to https://www.ffmpeg.org/download.html

```
libavutil      54. 31.100
libavcodec     56. 60.100
libavformat    56. 40.101
```

it should be >= 2.8.